react/ButtonPlain.js

```diff
@@ -0,0 +1,3 @@
+import ButtonPlain from './components/ButtonPlain/index'
```

Comment:
You can reduce this document by:
```suggestion
export {default}  from './components/ButtonPlain/index'
```